backend/internal/server/services/auth/roles.go

```diff
@@ -51,3 +45,65 @@ func HasCorrectRole(usersRoles []Role, roles []Role) bool {
 
 	return false
 }
+
+type Roles []Role
+
+var (
+	Public        Roles = []Role{None}
+	Authenticated Roles = []Role{Manager, Referee, ScoreKeeper, Captain, Player}
+	Staff         Roles = []Role{Manager, Referee, ScoreKeeper}
+	ManagerOnly   Roles = []Role{Manager}
+)
+
+// func (rs *Roles) Scan(value interface{}) error {
```

Comment:
Is this needed? If not, can these commented out lines be removed?

Reply:
removed comments

Reply:
removed comments